src/transformStateWithClones.js

```diff
@@ -7,7 +7,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const arr = [];
+  let newState = JSON.parse(JSON.stringify(state));
+
+  for (const obj of actions) {
```

Comment:
please give your variables discriptive names -> ` action of actions` would be better